model/feature-flag/logs.yaml

```diff
@@ -1,11 +1,49 @@
 groups:
-  - id: log-feature_flag
+  - id: feature_flag.evaluation
     type: event
-    stability: experimental
-    name: feature_flag
+    name: feature_flag.evaluation
     brief: >
-      This event describes feature flag evaluation representation on a Log Record.
+      This semantic convention defines the attributes used to represent a
+      feature flag evaluation as an event.
+      A `feature_flag.evaluation` event SHOULD be emitted whenever a feature flag
+      value is evaluated, which may happen many times over the course of an
+      application lifecycle.
+      For example, a website A/B testing different animations may evaluate a
+      flag each time a button is clicked.
+      A `feature_flag.evaluation` event is emitted on each evaluation even if the result is the same.
     attributes:
       - ref: feature_flag.key
-      - ref: feature_flag.provider_name
+        requirement_level: required
       - ref: feature_flag.variant
+        requirement_level:
+          conditionally_required: If feature flag provider supplies a variant or equivalent concept.
+      - ref: feature_flag.system
+        requirement_level: recommended
+      - ref: feature_flag.context.id
+        requirement_level: recommended
+      - ref: feature_flag.version
+        requirement_level: recommended
+      - ref: feature_flag.set.id
+        requirement_level: recommended
+      - ref: feature_flag.evaluation.reason
+        requirement_level: recommended
+      - ref: feature_flag.evaluation.error.code
```

Comment:
any reason not to reuse generic `error.type`? It'd be great for consistency, especially if we'll ever have some FF metrics or spans.
It seems you'd still have `feature_flag.evaluation.error.message` for anything FF-specific.